libsolidity/ast/CallGraph.h

```diff
@@ -61,8 +61,8 @@ struct CallGraph
 	/// any calls.
 	std::map<Node, std::set<Node, CompareByID>, CompareByID> edges;
 
-	/// Contracts that may get created with `new` by functions present in the graph.
-	std::set<ContractDefinition const*, ASTNode::CompareByID> createdContracts;
+	/// Contracts that need to be compiled before this one can be compiled.
```

Comment:
Please explain the value of the map - is it one of the AST nodes that create the dependency?